src/v1.9.0/preExistingConditions.ts

```diff
@@ -0,0 +1,167 @@
+import { ScorableCollection } from './types';
+
+export type PreExistingCondition =
+  | 'diabetesType2'
+  | 'heartDisease'
+  | 'cancer'
+  | 'kidneyDisease'
+  | 'copd'
+  | 'organTransplant'
+  | 'sickleCellDisease'
+  | 'overweight'
+  | 'asthma'
+  | 'smoke'
+  | 'steroids'
+  | 'pregnant'
+  | 'cerebrovascular'
+  | 'hypertension'
+  | 'diabetesType1'
+  | 'lungDisease'
+  | 'thalassemia'
+  | 'neurologicCondition'
+  | 'liverDisease'
+  | 'compromisedImmuneSystem'
+  | 'hiv'
+  | 'boneMarrowTransplant';
+
+export const preExistingConditions: ScorableCollection<PreExistingCondition> = {
+  key: 'preExistingConditions',
+  label: 'Select pre-existing conditions:',
+  items: [
+    {
+      key: 'diabetesType2',
+      prompt: 'Type 2 diabetes mellitus',
+      aliases: ['Do you have diabetes?'],
+      preExisting: 20,
+    },
+    {
+      key: 'heartDisease',
+      prompt:
+        'Heart conditions, such as heart failure, coronary artery disease, or cardiomyopathies',
+      aliases: [
+        'Do you have heart disease, heart failure, or coronary artery disease?',
+      ],
+      preExisting: 20,
+    },
+    {
+      key: 'cancer',
+      prompt: 'Cancer',
+      aliases: ['Are you on chemotherapy or chemoradiation?'],
+      preExisting: 20,
+    },
+    {
+      key: 'kidneyDisease',
+      prompt: 'Chronic kidney disease',
+      aliases: ['Are you on hemodialysis?'],
+      preExisting: 20,
+    },
+    {
+      key: 'copd',
+      prompt: 'COPD',
+      aliases: [
+        'Are you a smoker, have asthma, cystic fibrosis, COPD, or emphysema?',
+      ],
```

Comment:
I could be confused about how aliases are used but wanst the point of this iteration to unbundle some of these questions? Should the alias for COPD still be asking about  asthma, cystic fibrosis, COPD, or emphysema?

Reply:
Yeah great question. So the goal of aliases is to be backwards compatible - Allowing past version to be scored under the current version. We felt, for the grouped previous answers, we should pair them with one of the new options and to choose the highest scoring one. There is no way to know if past users were picking it for one or many and we don't know if it was for the lightest or the weakest but when possible we paired it with the option that had the same score and was worst of the group. 

Not sure if that makes sense but we couldn't think of a better answer.